client/src/cmdlfnexwatch.c

```diff
@@ -257,12 +281,90 @@ static int CmdNexWatchReader(const char *Cmd) {
     }
 
     do {
-        lf_read(false, 20000);
+        lf_read(true, 20000);
         demodNexWatch(!cm);
     } while (cm && !kbd_enter_pressed());
     return PM3_SUCCESS;
 }
+//TOCHANGE
+
+static unsigned int bin2int(unsigned char *b, int size) {
+    unsigned int val = 0;
+    for (int i = 0; i < size; i++) {
+        if (b[i] == '1') {
+            val += pow(2, size - 1 - i);
+        }
+    }
+    return val;
+}
+static BYTE xor_parity(BYTE *stream_, int bit_number) {
```

Comment:
there is parity functions in client/src/utils.c  
if that isn't suitable,  consider returning a uint8_t with 0 or 1, instead of unsign char.